components/server/src/ide-service.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { JetBrainsConfig, TaskConfig, Workspace } from "@gitpod/gitpod-protocol";
```

Comment:
nit: ideally we had a top-level package `ide` to move the other stuff into. But that's a clear follow-up.